Sources/AudioKit/Nodes/Node.swift

```diff
@@ -65,34 +65,35 @@ open class Node {
         }
     }
 
-    func makeAVConnections() {
+    public func makeAVConnections() {
```

Comment:
Why is this public?

Reply:
sorry, the sample rate changes got mixed into the other access experiments. that doesn't need to be. though subclassing Node from outside of AudioKit may have some challenges.

let me quarantine just the sample rate thing